libs/dh/market-participant/actors/feature-delegation/src/lib/util/dh-group-delegations.ts

```diff
@@ -25,7 +25,7 @@ export function dhGroupDelegations(delegations: DhDelegations): DhDelegationsByT
     const index = groups.findIndex((group) => group.type === delegation.process);
 
     if (index === -1) {
-      groups.push({ type: delegation.process, delegations: [delegation] });
+      groups.push({ type: delegation.process, delegations: [delegation], totalCount: 0 });
```

Comment:
question: is `totalCount` used somewhere?

Reply:
The type had it, and it began to show an error, so i just added it without checking. But maybe not

Reply:
Not used, removed now